.github/workflows/build.yml

```diff
@@ -14,51 +14,30 @@ jobs:
   tests:
     strategy:
       fail-fast: false
-      max-parallel: 4
+      max-parallel: 5
       matrix:
         platform: [ubuntu-latest] #, windows-latest]
-        python-version: [3.8, 3.9, '3.10', '3.11', '3.12']
+        python-version: [3.8, 3.9, '3.10', 3.11]
         include:
-          - python-version: 3.8
-            tox-env: py38
-            continue-on-error: False
-          - python-version: 3.9
-            tox-env: py39
-            continue-on-error: False
-          - python-version: '3.10'
-            tox-env: py310
-            continue-on-error: False
-          - python-version: '3.11'
-            tox-env: py311
-            continue-on-error: False
-          - python-version: '3.12'
-            tox-env: py312
-            continue-on-error: False
+          - python-version: 3.12
+            continue-on-error: false
```

Comment:
This can be removed when we are confident about Python 3.12 release candidates or when it is production released in three weeks.

Reply:
Sorry, see my global comment below. Tests don't run if only one is defined. I don't know why, but it is just how GitHub works.

Reply:
Please add back all of these. I'm just going to have to do it later when I need to alter the setup.

```yaml
        include:
          - python-version: 3.8
            tox-env: py38
            continue-on-error: False
          - python-version: 3.9
            tox-env: py39
            continue-on-error: False
          - python-version: '3.10'
            tox-env: py310
            continue-on-error: False
          - python-version: '3.11'
            tox-env: py311
            continue-on-error: False
          - python-version: '3.12'
            tox-env: py312
            continue-on-error: False
```